ui-admin/src/study/StudyEnvironmentRouter.tsx

```diff
@@ -91,7 +91,7 @@ function StudyEnvironmentRouter({ study }: {study: Study}) {
       <Route path="participants/*" element={<ParticipantsRouter studyEnvContext={studyEnvContext}/>}/>
       <Route path="kits/*" element={<KitsRouter studyEnvContext={studyEnvContext}/>}/>
       <Route path="siteContent" element={<SiteContentLoader portalEnvContext={portalEnvContext}/>}/>
-      <Route path="images" element={<SiteImageList portalContext={portalContext} portalEnv={portalEnv}/>}/>
+      <Route path="media" element={<SiteImageList portalContext={portalContext} portalEnv={portalEnv}/>}/>
```

Comment:
Normally I wouldn't be so quick to change a URL but this doesn't seem like a mission-critical path

Reply:
concur, I just did a quick check of user docs/smoke test checklists and this was not referenced